velox/common/testutil/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_library(test_util TestValue.h)
+target_link_libraries(test_util ${FOLLY_WITH_DEPENDENCIES})
```

Comment:
you probably need 
if(${VELOX_BUILD_TESTING})
  add_subdirectory(tests)
endif()

if you have tests directory under this..